guides/common/modules/proc_registering-project-as-a-keycloak-client.adoc

```diff
@@ -14,11 +14,12 @@ You can also register two different {Project} clients to {Keycloak} if you want
 
 .Procedure
 
+// python3-lxml is only needed because of https://issues.redhat.com/browse/RHEL-31496
```

Comment:
Let's make it clear that EL 9 is unaffected
```suggestion
// python3-lxml is only needed on EL8 because of https://issues.redhat.com/browse/RHEL-31496
```